### ISSUE TITLE
Bug report: write_map omits line ending for the last key-value pair

### ISSUE THREAD

**Issue by SHuang-Broad** [broadinstitute/cromwell#7021]
Like reported in the title, `write_map` will write a TSV without a line ending `\n` for the last pair.

Minimal example
This will yield 1, as opposed to the expected 2.

```wdl
workflow TestWriteMap {
    call WriteAMap {
        input: input_map = {'A': 1, 'B': 2}
    }
    output {
        Int tsv_len = WriteAMap.tsv_len
    }
}

task WriteAMap {
    input {
        Map [String, String] input_map
    }
    command <<<
        set -eux

        two_col_tsv=~{write_map(input_map)}
        wc -l "${two_col_tsv}" | awk '{print $1}' > "map_tsv_len.txt"
    >>>
    output {
        Int tsv_len = read_int("map_tsv_len.txt")
    }
    runtime {docker: "gcr.io/cloud-marketplace/google/ubuntu2004:latest"}
}
```

**Comment by SHuang-Broad** [comment]
Sorry that I forgot to include Cromwell versions.
I've tested on a local Cromwell server, version 63, and on Terra. Both give the same behavior.

**Comment by SHuang-Broad** [comment]
closed by #7022

**Comment by cjllanwarne** [comment]
Closed by #7022
